### ISSUE TITLE
New home?

### ISSUE THREAD

**Issue by mavit** [jdecked/twemoji#10]
Hello, I maintain the [Fedora package of Twemoji](https://packages.fedoraproject.org/pkgs/twitter-twemoji-fonts/twitter-twemoji-fonts/).  Is this repository likely to me the permanent new home of the project?  Should I update all our links to point here?

Thanks!

**Comment by jdecked** [comment]
Hello @mavit, indeed this is the new home as far as everyone who last worked on Twemoji is concerned, so I'd update the links. We have no information on whether Twitter will continue to maintain the official package; it seems unlikely.

**Comment by OfficialCRUGG** [comment]
@jdecked Could you further elaborate on how this repo is related to the original Twemoji? I'm a bit confused right now..

**Comment by jdecked** [comment]
@OfficialCRUGG sure. The original Twemoji was maintained by 2 Twitter employees, @sofodesign and me. With Elon Musk's acquisition of Twitter, we both no longer work there, nor do we have commit permissions to the original repo anymore. There's no indication at all from Twitter's side whether they intend to keep maintaining the original Twemoji; given Sofo and I did it in essentially our free time, we imagine it's highly unlikely and probably pretty low on the priority list of anyone there right now. We (and other past Twitter employees who worked on Twemoji and left well before the Musk acquisition) don't want Twemoji to go to the big Internet graveyard in the sky, so we've forked the original Twemoji here and plan to continue working on it (releasing new Unicode-approved emoji, maintenance, etc) despite no longer being employed by Twitter. I hope that clarifies things; if not, feel free to reply with any further questions.
